tune_the_model/cli.py

```
@@ -412,7 +420,10 @@ def classify(self, input: str):
 
     @inited
     def wait_for_training_finish(self, sleep_for: int = 60):
+        logger.info("We are taking your model in work")
         while self.status is not TuneTheModelStatus.READY:
+            if (self.status is TuneTheModelStatus.FAILED):
+                logger.error("Fit failed, something went wrong. Please contact us about this issue")

```

I think we should raise an exception here.